### PR TITLE
Psp fitting fix branch

### PR DESCRIPTION
**Pull request by corinneteeter**
This version includes tests not yet migrated to neuroanalysis.  However the test data has been deleted from the history.  I still did not figure out how to access the data in the submodule so the path is currently hard coded.  Please suggest the fix or just merge.

### CODE REVIEW

**Comment by corinneteeter** [comment]
I reverted code so that exp_amp is a string to be compatible with the current FitModel.make_params api.  Please merge.

File: analyses/psp_fitting.py [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+parser = argparse.ArgumentParser(description='Enter organism and type of connection you"d like to analyze ex: mouse ee (all mouse excitatory-'
+                'excitatory). Alternatively enter a cre-type connection ex: sim1-sim1')
+parser.add_argument('--organism', dest='organism', help='Select mouse or human')
+parser.add_argument('--connection', dest='connection', help='Specify connections to analyze')
+args = vars(parser.parse_args(sys.argv[1:]))
+
+# get experiments from cache
+all_expts = cached_experiments()
+
+#if args['organism'] == 'mouse':
```
Comment (campagnola) [inline analyses/psp_fitting.py]: just dump the commented code.

File: analyses/tests/test_psp_fitting.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
+from multipatch_analysis.connection_detection import create_all_fit_param_combos, fit_psp
+import os
+import numpy as np
+from pprint import pprint
+import json
+import neuroanalysis.data
+
+test_data_dir = os.path.join(os.path.dirname(neuroanalysis.__file__), '..', 'test_data', 'test_psp_fit')                                                                                                                                      
+
+def test_create_all_fit_param_combos():
```
Comment (campagnola) [inline analyses/tests/test_psp_fitting.py]: move this to multipatch_analysis/tests/test_connectino_detection

File: multipatch_analysis/synaptic_dynamics.py [hunk 3]
```diff
@@ -520,6 +520,20 @@ def plot_model_results(self, model=None, fit=None):
         return rel_plots
 
     def cross_talk(self):
+        """#TODO: Luke you should edit this
```
Comment (campagnola) [inline multipatch_analysis/synaptic_dynamics.py]: Let's not commit TODOs

File: analyses/psp_fitting.py (hunk 1 above)
Comment (corinneteeter) [reply analyses/psp_fitting.py]: removed entire file.

File: analyses/tests/test_psp_fitting.py (hunk 2 above)
Comment (corinneteeter) [reply analyses/tests/test_psp_fitting.py]: done.

**Comment by corinneteeter** [comment]
Resolved changes:)  Please merge.

**Comment by campagnola** [comment]
why does it say it's trying to merge 375 commits?

**Comment by corinneteeter** [comment]
Hmm, I am not sure.  I suppose that potentially could have happened when I tried to filter my repository to get rid of the test data that was originally in there...  I think that process might rename commits but I am not sure.  I guess one potential option is to wipe out all previous commits.  Can we do that?

**Pull request merged**